codec-http2/src/main/java/io/netty/handler/codec/http2/DefaultHttp2RemoteFlowController.java

```diff
@@ -415,12 +418,13 @@ public boolean visit(Http2Stream child) throws Http2Exception {
     }
 
     /**
-     * The outbound flow control state for a single stream.
+     * The remote flow control state for a single stream.
      */
-    private final class FlowState {
+    private final class DefaultFlowState implements RemoteFlowControlState {
         private final Deque<FlowControlled> pendingWriteQueue;
         private final Http2Stream stream;
         private int window;
+        private int initialWindowSize;
```

Comment:
@nmittler - This is to support `unavailableWindow`.  This has no functional use in the codec the way the flow controllers are implemented today. I don't have a concrete use case for it but seems like it is useful...I'm debating taking it out though...